packages/babel-plugin-transform-classes/test/fixtures/get-set-loose/call-semantics-getter-defined-on-parent/exec.js

```diff
@@ -20,11 +20,14 @@ class Obj extends Base {
     super.test(...[1, 2, 3]);
     return super.test(...arguments);
   }
-
-  test() {
+}
+Object.defineProperty(Obj.prototype, 'test', {
```

Comment:
I don't understand what caused this change. Any idea? It doesn't seem like this should be split off like this.

Reply:
This is the reason I filled #7771. https://github.com/jridgewell/babel/commit/ab403f3d6d768a98130ef3588feed71b941beaac#diff-0cd85dd4813df0687b8938547953d585L24

`test` here will be set on the subclass, but the super defines a `get test`, so it'll throw.

Reply:
Ahh got it.